sdk/python/kfp/compiler/compiler.py

```diff
@@ -484,15 +498,19 @@ def _group_to_dag_template(self, group, inputs, outputs, dependencies):
           else:
             param_name = '%s-%s' % (
               sanitize_k8s_name(pipeline_param.op_name), pipeline_param.name)
-            withparam_value = '{{tasks.%s.outputs.parameters.%s}}' % (
+            need_propagating_param = not [g.name for g in sub_groups if g.name == pipeline_param.op_name]
+            if group.type == 'subgraph' and need_propagating_param:
```

Comment:
Sorry, I don't fully understand the logic here.
Can you maybe help me with an example where `group.type == 'subgraph'` but `need_propagating_param` is False?

Reply:
Yeap, it's redundant..